sources/nodes/kube.go

```diff
@@ -78,8 +78,15 @@ func (self *kubeNodes) List() (*NodeList, error) {
 			nodeList.Items[Host(node.Name)] = Info{PublicIP: node.Status.HostIP, InternalIP: addrs[0].String()}
 			goodNodes = append(goodNodes, node.Name)
 		} else {
-			glog.Errorf("Skipping host %s since looking up its IP failed - %s", node.Name, err)
-			self.recordNodeError(node.Name)
+			ip := net.ParseIP(node.Name)
```

Comment:
I think we can split the pull request because the first changeset (	resource := filepath.Join("stats", pod.Namespace, pod.Name, pod.ID, containerName)) is required if you have a kubernetes cluster with multiple namespaces. Maybe i can find a better solution for the hostname issue.